packages/perseus/package.json

```diff
@@ -61,6 +61,7 @@
         "@khanacademy/wonder-blocks-dropdown": "5.3.0",
         "@khanacademy/wonder-blocks-form": "^4.7.1",
         "@khanacademy/wonder-blocks-icon": "4.1.0",
+        "@khanacademy/wonder-blocks-icon-button": "^5.3.3",
```

Comment:
Should have added this to peerDependencies as well. Will be added in the next Perseus release.